### ISSUE TITLE
help: How can I import runtime config without using `useContext`

### ISSUE THREAD

**Issue by yulafezmesi** [nuxt-community/composition-api#619]
**📚 What are you trying to do? Please describe.**

Trying to import public config file without using useContext hook.

```javascript
 const { $config } = useContext()
```

**🔍 What have you tried?**

According to Nuxt3 official document, I've seen [this](https://v3.nuxtjs.org/docs/usage/runtime-config/#api-routes) kind of usage that is don't have to use useContext hook to import public config file. Is there any way to do that in composition api?

```javascript
 import config from '#config'
```

**ℹ️ Additional context**
Add any other context or information.

**Comment by danielroe** [comment]
I'd highly recommend migrating to Nuxt Bridge and you'll be able to use `useRuntimeConfig()`.

**Comment by yulafezmesi** [comment]
@danielroe  unfortunately,  It's not possible in near future :) Is there any tricky way to handle it?

**Comment by danielroe** [comment]
No. The example you reference is using a feature of Nuxt Nitro, which is only available with Bridge.
